source/StartConditionsPanel.cpp

```diff
@@ -0,0 +1,253 @@
+/* StartConditionsPanel.cpp
+Copyright (c) 2020 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "StartConditionsPanel.h"
+
+#include "Color.h"
+#include "Command.h"
+#include "ConversationPanel.h"
+#include "DataFile.h"
+#include "Dialog.h"
+#include "Files.h"
+#include "FillShader.h"
+#include "Font.h"
+#include "FontSet.h"
+#include "Format.h"
+#include "GameData.h"
+#include "GameWindow.h"
+#include "Information.h"
+#include "Interface.h"
+#include "LoadPanel.h"
+#include "MainPanel.h"
+#include "Messages.h"
+#include "PlayerInfo.h"
+#include "Planet.h"
+#include "Preferences.h"
+#include "Rectangle.h"
+#include "Ship.h"
+#include "ShipyardPanel.h"
+#include "StarField.h"
+#include "System.h"
+#include "UI.h"
+#include "WrappedText.h"
+
+#include <SDL2/SDL.h>
+
+#include "gl_header.h"
+
+using namespace std;
+
+namespace 
+{
+    const double LIST_ITEM_SIZE = 20.;
+}
+
+StartConditionsPanel::StartConditionsPanel(PlayerInfo &player, UI &gamePanels, LoadPanel *loadPanel)
+    : player(player), gamePanels(gamePanels), loadPanel(loadPanel)
+{
+    if(GameData::Start().size())
+    {
+        chosenStart = GameData::Start()[GameData::Start().size()-1];    
+    }
+    else
+    {
+        chosenStart = nullptr;
+    }
+    
+}
+
+
+void StartConditionsPanel::Draw()
+{
+    glClear(GL_COLOR_BUFFER_BIT);
+
+    Information info;
+
+    string descriptionText; // String that will be shown in the description panel
+
+    if(chosenStart){
```

Comment:
I'd very much prefer this to be an unconditional path, i.e. _always_ display the details of the selected start. When there isn't a user-selected start, the selected start is e.g. a customized start condition that has all the default strings & info.

Reply:
The last start in the list is automatically selected in the constructor anyways. The only way chosenStart can be null is when there are no `start`s defined